Post hackathon (#25)
kdesai2018/ultimate-hackathon-starting-guide: post-hackathon/README.md

# Post-Hackathon 

> Congrats you made it through the hackathon and you now have some working project! What should you do? Definitely DO NOT just make your project a history. Here are the steps that you should follow to make your experience worth your effort. 

Reviewer: ```suggestion
Congratulations! You made it through the hackathon and you have a (hopefully) working project! What should you do next? Definitely DO NOT just make your project a history. Here are the steps that you should follow to make your experience worth your effort. 
```

## 1. Post-Mortem

### The Positives

> Make a list of everything that went right in this project so that it can be repeated next time. From the small-scale accomplishments to the large-scale accomplishments, each detail counts and is probably worth noting! This makes it easier next time when you want to add to the project, develop a similar project with more functionalities, or use it as reference in an internship, job, and/or another experience of applying the knowledge.

Reviewer: ```suggestion
Make a list of everything that went right in this project so that it can be repeated next time. From the small-scale accomplishments to the large-scale accomplishments, each detail counts and is probably worth noting! This makes it easier next time when you want to add to the project, develop a similar project with more functionalities, or use it as reference in any kind of interview.
```

### The Negatives

> Yes, with positives must also come a list of everything that went wrong and should be avoided next time. A suggestion is if something during the project did not go as planned, then make note of it and detail the solutions (if there are any) or what was the root of the problem. 

Reviewer: ```suggestion
Yes, with positives must also come a list of everything that went wrong and should be avoided next time. A suggestion is if something during the project did not go as planned, then make note of it and detail the solutions (if there are any) as well as the root of the problem(s). 
```

In the future if this project is ever referenced or developed again, the post-mortem will save you plenty of time!

## 2. Cost-Benefit Analysis
Evaluate your project overall and create a comparison between the benefits that came from the experience in contrast to the negative experiences. What did you learn from the hackathon, what do you look forward to, and what could have made the experience more worthwile?

If you worked on the project with a team, discuss with each other what each one liked and disliked about the event. Additionally, it is important to have an understanding of what is to come of this product, and what the future plans are regarding it.

## 3. Blog / Recount your experience

> Congrats, you know exactly what is to become of your project and what your future plans are regarding its scope! Recount your experience to others, tag the host (us Freetail Hackers) on your social media, and let us know how we did or how we could have done better! Or, if you just want to maintain a lower profile, write an article talking about your product and its functionalities. You do not need to publish it, but iwriting it is great way to wrap up the hackathon and see how your hardwork payed off!

Reviewer: ```suggestion
Good work! You know exactly what is to become of your project and what your future plans are regarding its scope! Recount your experience to others, tag the hackathon organizers (we are @freetailhackers) on your social media, and let us know how they did and/or what they could have improved on! Another option is to write an article talking about your product and its functionalities. A lot of employers like seeing posts like these on sites like LinkedIn, since they demonstrate your ability to build functional products with a team in a stressful situation. 

If you want to keep a lower profile, you do not need to publish your article. Just writing it is great way to wrap up the hackathon and see how your hard work paid off!
```
